indirect_test.cc

```diff
@@ -54,6 +55,51 @@ TEST(IndirectTest, ValueAccessFromDefaultConstructedObject) {
   EXPECT_EQ(*a, 0);
 }
 
+template <typename Allocator = std::allocator<void>>
+struct AllocatorAwareType {
+  using allocator_type = typename std::allocator_traits<
+      Allocator>::template rebind_alloc<AllocatorAwareType>;
+
+  AllocatorAwareType() = default;
+  AllocatorAwareType(std::allocator_arg_t, const allocator_type& alloc)
+      : children(alloc) {}
+
+  std::vector<AllocatorAwareType, allocator_type> children;
+};
+
+template <typename T>
+struct DefaultConstructibleTrackingAllocator : xyz::TrackingAllocator<T> {
+  using xyz::TrackingAllocator<T>::TrackingAllocator;
+
+  unsigned dummy = 0;
```

Comment:
Nit: maybe go for `unused`? `dummy` was in a non-inclusive language list or two.